config/samples/ocs.openshift.io_v1alpha1_volumereplicationclass.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: ocs.openshift.io.ocs.openshift.io/v1alpha1
+kind: VolumeReplicationClass
+metadata:
+  name: volumereplicationclass-sample
+spec:
+  # Add fields here
+  foo: bar
```

Comment:
```suggestion
```

Reply:
Would like to keep it as is in the first commit. Will remove in the next PR.